tracer/build/_build/Build.Profiler.Steps.cs

```diff
@@ -151,7 +155,7 @@ partial class Build
     Target PublishProfilerLinux => _ => _
         .Unlisted()
         .OnlyWhenStatic(() => IsLinux)
-        .After(CompileProfilerNativeSrc)
+        .After(CompileProfilerNativeSrcAndTests)
```

Comment:
I'm thinking we probably still don't want to bundle these two steps together... my thinking is that _locally_ we don't want to _force_ people to run the native tests (even though they're generally quick) - we should probably keep these as a separate stage, and just call the extra test step in the pipeline